assets/css/custom/program.scss

```diff
@@ -741,6 +511,31 @@ input[type='number'] {
 }
 .sonata-admin-thumbnail-image
 {
-    height: 96px;
-    width: 96px;
-}
\ No newline at end of file
+  width: 96px;
+  height: 96px;
+}
+.material-icons.md-30 { font-size: 30px; }
```

Comment:
better not define material icon specifics in the project sheet, better move it to something like icons, or material. to keep em consistent on all pages.